performant-software/Annotation-Studio
# Added HTML download for snapshots

#### What's this PR do?
Makes it easy to export HTML documents from AS, for import to COVE editions, or elsewhere.
Closes #3 

#### How to test
- Visit a document
- Annotate
- Click Snapshot at bottom of the the filter sidebar
- Click Preview at bottom of the the filter sidebar
- Click Download from the preview page sidebar
- Double click the downloaded file -- should open either in a browser as a web page or in an editor as an HTML file, depending on your OS settings.
